src/Runner.php

```diff
@@ -30,6 +30,7 @@ public function __construct(Container $container = null)
         $commands_directory = __DIR__ . '/Commands';
         $top_namespace = 'Pantheon\Terminus\Commands';
         $this->commands = $this->getCommands(['path' => $commands_directory, 'namespace' => $top_namespace,]);
+        $this->commands[] = 'Pantheon\\Terminus\\Authorizer';
```

Comment:
It would be possible to name this 'AuthorizerCommand' and put it in the commands directory, then it would technically work, but the filename would be deceiving, because it is a hook, not a command.

If there were going to be more of these, then you could search for *Hook.php in the Hooks directory, and discover them.  I just listed it explicitly since there was only one. I can change the mechanism to something else if desired.